subjects/popint/README.md

```diff
@@ -0,0 +1,39 @@
+## popint
+
+### Instructions
+
+Write a function that receives a slice of int and returns a new slice without the last element.
```

Comment:
just missing explaining what happens when the slice is empty

Reply:
I'll add that. Thank you.